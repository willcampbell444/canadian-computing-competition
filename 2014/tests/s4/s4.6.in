100
13000000
0 0 1000000000 1000000000 307634
728593843 524947301 926449225 575667018 720370
445531249 561203433 854394645 735117866 904001
263480141 114500437 308451243 199432867 689543
164695809 83507719 645902059 146053222 676827
170713556 144566494 377664530 239073757 43095
185028068 120709472 565160225 210401214 103930
213253187 143573905 387962668 728191978 976071
663362410 189908225 679995005 915546541 821327
128044534 681493744 578737547 851842607 199879
4325575 539155848 103923983 832961867 182455
236656793 184135218 580437280 350988901 347639
221495059 321424258 619906266 871274940 110580
206254315 477649141 555223495 824817444 823117
96018532 23231620 159175994 653773836 792165
76414861 902326500 113706653 954737981 437227
727380293 4319293 849900194 249160659 67357
421276468 572125042 815946315 932992096 33919
141084403 105698209 977717802 176593511 796558
86810696 355300655 734030813 885954553 553119
328522933 304461688 781526255 894761311 352615
544596964 465853628 950314422 891586393 15675
32010497 922489907 490464438 986550858 914188
2981703 278685776 202596554 514337018 560930
176658014 268824337 827590455 854174145 17250
186985243 78301213 784987051 685382118 953297
99733936 127577761 636645334 629508009 239738
469648210 375045938 967240554 753761867 554600
600791416 33203712 856506204 509863365 802604
167443113 805801199 924487597 818839928 852190
165520086 229378819 319702117 697741115 647762
292368870 400996948 787297329 459442529 3944
379651732 425723065 989007407 724044863 20230
19615859 406302364 822256833 911722800 812867
659701204 833643722 904232043 933044746 438822
138688777 361008361 826203746 377080714 301095
17492322 374170178 639041808 602926987 518895
109966369 3409237 912615674 368810556 188029
188140975 408757562 982680580 989421635 523918
65910957 171036238 595620416 249866410 201564
98440130 837444823 789257509 920839914 300917
190931711 176546698 240875245 648681353 490893
441136427 104254288 540733479 859357723 219318
33311125 780048403 537588134 952819972 696919
39330122 389375757 855658684 504984869 298918
523951126 9116914 840807521 802586765 617855
225598291 114818459 752015724 186993866 308880
445569725 207869564 500847058 748074406 767036
878874614 162218097 984071416 646092010 791430
169596481 490235176 280828072 855816838 107239
148959489 743666061 233640265 984805717 196769
28226044 387209484 950110942 619181681 325757
465108368 79412812 808310108 235849090 765076
339302580 272828221 683359457 882357853 347768
25507761 5188306 300496255 351998620 825696
88523492 755629513 659380138 979339424 915212
16238302 203196495 602347125 772692508 471314
346994254 514308566 404261420 576405411 368166
221293947 638669204 725612982 965224001 27116
453965617 938193968 682424850 951762163 578834
166398723 432607645 250039352 933358725 28319
593997653 691927052 802712216 814545908 804710
183298625 208423770 477929408 970703670 452712
138020173 169622707 815539047 215687429 201646
39623576 26489840 478604529 225555820 12979
146928820 225932420 301617621 619462269 73623
18203193 411788284 348393659 436877194 873486
437649041 435924306 513074218 884517799 821838
140800556 347098443 265081704 493939661 924828
279520428 110472149 295626504 146075984 228531
370086779 216784814 545604330 728831519 595924
174913154 7733374 564288315 705908247 224495
330177099 188742239 667571232 415107489 447448
107481070 193574941 769286761 551020380 19375
322494491 48474980 939088493 63652048 669841
102381597 269922524 112281494 386078110 675762
16546905 78425065 681609134 919920618 86776
563905576 365990776 712544465 508660199 785803
328465305 128903277 642113190 891041349 843557
165578520 532357707 738276527 846055502 91341
187614545 72619732 318278510 731647213 329363
77892802 765272508 655463267 927486199 731599
629081215 840337512 970614990 905969518 84706
53713600 492393013 602096469 614322038 860794
493857299 28307216 958471903 555416152 831734
593209849 458825962 667145536 702885759 392560
361104449 86666499 985676803 374153641 914606
264447092 204963898 562805822 418592794 639765
244285969 373495508 855974681 873014759 756788
199080945 685022864 665542921 966899168 422260
76368676 126095205 863217686 493108049 568964
39066324 27930276 574912648 43685449 692265
493591276 839686360 869243646 852225950 83001
13130120 73085162 707962650 668837363 650229
249958400 236348053 500513882 572798462 28174
313722415 419298039 510107755 453254098 274245
644958083 106029504 773913558 649698006 508688
156909059 592807934 511080765 853614676 291499
451091822 460069747 645409650 675957022 154703
496555165 274184491 511812354 358210636 475290
