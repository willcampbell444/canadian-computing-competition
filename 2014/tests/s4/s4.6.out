6746915871404074
